### DISCUSSION TITLE
Can Budibase support YugabyteDB

### DISCUSSION THREAD

**Discussion by seethb** [Budibase/budibase#14400] [Ideas]
Hi YugabyteDB is a open source distributed sql database and its highly compatible with Postgresql. We also have our own Smart Driver for different languages like JAVA, Python, Node JS, C#, Rust. Happy to contribute on integration with Budibase, Can you please share the instruction on how to add new source and will raise a PR post understanding of this integration process and coding expectations. Thanks.

**Comment by mikesealey** [comment]
Hey @seethb!

We love to hear about things like this! I think the first steps would be to look at the [Custom Datasource](https://docs.budibase.com/docs/custom-datasource) section of our documentation. You might also find help, support, suggestions, and ideas in our [community discord server](https://discord.gg/ZepTmGbtfF). We have a [list of custom plugins](https://github.com/Budibase/plugins) you could also take a look at, which may help get you off the ground. 

Looking forward to seeing what you come up with!

**Reply by seethb** [reply]
Thanks @mikesealey sure we will take a look.
